### PR TITLE
Completed react-form-controls

### PR DESCRIPTION
**Pull request by K-Bowler**
image

### CODE REVIEW

File: react-form-controls/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  render() {
+    return (
+      <form>
+        <label htmlFor="username">Username:</label>
+        <input type="text" id='username' name='username' value={this.state.username} onChange={this.handleUsernameChange}/>
+        <label htmlFor="password">Password:</label>
+        <input type="password" id='password' name='password' value={this.state.password} onChange={this.handlePasswordChange}/>
+        <button onClick={this.handleSubmit}>Sign up</button>
```
Comment (scbowler) [inline react-form-controls/src/index.jsx]: Don't use `onClick` for submitting a form. Use the `onSubmit` event on the form element instead.
```JSX
<form onSubmit={this.handleSubmit}>
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but 1 update needed. Let me know if you have any questions.

File: react-form-controls/src/index.jsx (hunk 1 above)
Comment (K-Bowler) [reply react-form-controls/src/index.jsx]: image

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**